frontend/src/CSets.js

```diff
@@ -223,114 +298,82 @@ function ConceptSetsPage(props) {
       </div>)
 }
 
-//TODO: How to get hierarchy data?
-// - It's likely in one of the datasets we haven't downloaded yet. When we get it, we can do indents.
-function ConceptSet(props) {
-  let {cset} = props;
-  return (
-    // (isLoading && "Loading...") ||
-    // (error && `An error has occurred: ${error.stack}`) ||
-    // (isFetching && "Updating...") ||
-    // (data && <div style={{
-    (<div style={{
-        padding: '1px 15px 1px 15px',
-        margin: '5px 5px 5px 5px',
-        border: '5px 5px 5px 5px',
-        background: '#d3d3d3',
-        'border-radius': '10px',
-      }}>
-        <h4>{cset.conceptSetNameOMOP} v{cset.version}</h4>
-        <List>
-          {Object.values(cset.concepts).map((concept, i) => {
-            return <ListItem style={{
-              margin: '3px 3px 3px 3px',
-              background: '#dbdbdb',
-              'border-radius': '5px',
-              'font-size': '0.8em'
-            }} key={i}>
-              {concept.conceptId}: {concept.conceptName}
-            </ListItem>
-          })}
-        </List>
-      </div>)
-  )
-}
+// todo: move this back in, or accept cols as params
+// https://www.ag-grid.com/react-data-grid/column-definitions/
+const getColumnDefs = () => {
+  return [
+    { field: 'ConceptID' },
+    // { field: '23007370' },
+    // { field: '23600781' },
+  ];
+};
 
-function ConceptList(props) {
-  // http://127.0.0.1:8000/fields-from-objlist?objtype=OmopConceptSetVersionItem&filter=codesetId:822173787|74555844
+// TODO: Page state refresh should not be 'on window focus', but on autocomplete widget selection
+// TODO: Should we move comparison logic python side and do query at new backend_url?
```

Comment:
@Sigfried `// TODO: Should we move comparison logic python side and do query at new backend_url?`

I don't want to rely on Python too much. It's just a bit easier for me two wrangle things. But this wasn't so bad.